robottelo.properties.sample

```diff
@@ -188,3 +188,12 @@ ssh_key=
 # Parameter for deciding whether conduct initial sync or resync
 # 'resync' denotes resync; 'sync' denotes initial sync
 # sync_type='sync'
+
+# [libvirt_host]
```

Comment:
The main point here is, when we have more than one compute resource, like openstack and we just want to use libvirt, having all under compute_resources is good in terms of organization, but we may find problems with validation, like, in the example mentioned, don't knowing what should be required the libvirt_hostname and/or the openstack_hostname...